apps/mobile/src/features/send/screens/form.tsx

```diff
@@ -10,21 +10,102 @@ import { StxForm } from '@/features/send/forms/stx/stx-form';
 import { StxDataLoader } from '@/features/send/forms/stx/stx-loader';
 import { useSendNavigation, useSendRoute } from '@/features/send/navigation';
 import { useSendFlowContext } from '@/features/send/send-flow-provider';
-import { SendableAsset } from '@/features/send/types';
+import { Account } from '@/store/accounts/accounts';
 import { useSettings } from '@/store/settings/settings';
 import { analytics } from '@/utils/analytics';
 import { t } from '@lingui/core/macro';
 
+import { FungibleCryptoAsset } from '@leather.io/models';
 import { SheetRef } from '@leather.io/ui/native';
 
+import { Sip10Form } from '../forms/stx/sip10-form';
+import { Sip10DataLoader } from '../forms/stx/sip10-loader';
+
+function LoadedForm({
```

Comment:
Maybe this can be split into it's own container file now? `Form` ? then `Form` can be `FormLayout` or whatever?